peps/pep-0792.rst

```diff
@@ -0,0 +1,371 @@
+PEP: 792
+Title: Project status markers in the simple index
+Author: William Woodruff <william@yossarian.net>,
+        Facundo Tuesca <facundo.tuesca@trailofbits.com>
+Sponsor: Donald Stufft <donald@stufft.io>
+PEP-Delegate: Donald Stufft <donald@stufft.io>
+Discussions-To: https://discuss.python.org/t/79356
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 08-Apr-2025
+Post-History: `03-Feb-2025 <https://discuss.python.org/t/79356/>`__
+
+Abstract
+========
+
+This PEP proposes a standardized set of project status markers, as well
+as a mechanism for communicating those markers in the HTML and JSON
+simple indices.
```

Comment:
Can we disambiguate in the abstract to "distribution project" or the correct term in the context of package indices? I would tend to read "project status" here as relating to the source project, i.e. `pyproject.toml`.

Reply:
This could be my misunderstanding, but I thought "project" _was_ the disambiguating term: the PyPUG glossary defines both "distribution package" and "project," but the former appears to refer to a single instance (i.e. file) while the latter refers to the thing that gets packaged into distribution(s) and appears on PyPI.

Refs:

* https://packaging.python.org/en/latest/glossary/#term-Project
* https://packaging.python.org/en/latest/glossary/#term-Distribution-Package

(Apologies if I'm getting this wrong, though.)

Reply:
My view is that a "project" is nominally tied to a source tree / `pyproject.toml` (etc), and so when discussing some form of project status indicator, my expectation would be for it to relate to the project as a collection of sources. Note from the glossary "*that is intended to be packaged into a Distribution*".

The ideal here would be a term that captures 'project as a collection of distribution packages, as represented on a package index'. I recall some discussion with @pfmoore on the distinction between various types of 'projects' (in the context of PyPI pages), I'm not sure if any clarity was found there.

Reply:
I've gone with "index-supplied" as a stopgap, but happy to change further.